site/en/docs/web-platform/compute-pressure/index.md

```diff
@@ -1,264 +1,294 @@
-that receives periodic updates, allowing developers to observe changes in CPU
-utilization over time. A web app that uses this API can enable or
-disable heavyweight features in response to these
-observations. It's a feedback mechanism that unlocks new doors for adaptive
-behavior, which can in turn lead to smoother, cooler experiences across a range
-of hardware devices.
-
-## API status
-
-<div>
-
-| Step                                     | Status                                                                       |
-| ---------------------------------------- | ---------------------------------------------------------------------------- |
-| 1. Create explainer                      | [Complete](https://github.com/oyiptong/compute-pressure/blob/main/README.md) |
-| 2. Create initial draft of specification | [In Progress](https://github.com/WICG/compute-pressure/)                     |
-| 3. Gather feedback and iterate on design   | In progress                                                                  |
-| 4. Origin trial                          | Complete                                                                     |
-| 5. Launch                                | Not started                                                                  |
+  The Compute Pressure API offers high-level states that represent the pressure on the system.
+  It allows the implementation to use the right underlying hardware metrics to ensure that users
+  can take advantage of all the processing power available to them as long as the system is not
+  under unmanageable stress.
+
+## Current status
+
+<div class="table-wrapper scrollbar">
+
+| Step                                     | Status                              |
+| ---------------------------------------- | ----------------------------------  |
+| 1. Create explainer                      | [Complete][explainer]               |
+| 2. Create initial draft of specification | [Complete][specs]                   |
+| 3. Gather feedback & iterate on design   | [In progress](#feedback)            |
+| 4. Origin trial                          | Not Started                         |
+| 5. Launch                                | Not started                         |
 
 </div>
 
-## API Availability
+### Try out the Compute Pressure API
+
+To experiment with the Compute Pressure API locally, read this [page][how-to].
+
+## Use-Cases
+
+The primary use cases enhanced by the current Compute Pressure API are video conferencing and
+video games.
+
+These popular real-time applications are classified as soft. That is, the quality of service
+degrades if the system is exercised beyond certain states, but does not lead to a total system
+failure. These soft real-time applications greatly benefit from being able to adapt their workloads
+based on CPU consumption/pressure.
+
+Specifically, v1 aims to facilitate the following adaptation decisions for these use cases:
```

Comment:
```suggestion
Specifically, version one aims to facilitate the following adaptation decisions for these use cases:
```

Reply:
I will remove that "version one" all together.